cdk.json

```diff
@@ -36,9 +36,7 @@
       "PORT": "3838",
       "COST_CENTER": "NO PROGRAM / 000000",
       "STACK_NAME_PREFIX": "dca",
-      "HOST_NAME": "dca.dnt-dev.sagebase.org",
-      "HOSTED_ZONE_NAME": "dnt-dev.sagebase.org",
-      "HOSTED_ZONE_ID": "Z0441843EAHKE7DR12AX",
+      "ACM_CERT_ARN": "arn:aws:acm:us-east-1:878654265857:certificate/a3b7c804-c1fc-4e58-bbc6-541ef2d65816",
```

Comment:
this is a reference to `DCAProdAccount` however dev context is deployed to `DnTDevAccount`.  we will probably want to pass in a cert from DnTDevAccount.

Also can we add some documentation to describe how this works with what you added in Sage-Bionetworks-IT/organizations-infra?

Reply:
whoops, thanks, apparently its very easy to copy+paste the wrong ARN